### DISCUSSION TITLE
Frontend setup in expense Project

### DISCUSSION THREAD

**Discussion by palsachennaiah** [learndevopsonline/learndevopsonline#1582] [Q&A] [answered]
![Screenshot 2023-11-08 163120](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/2d3166a9-77bb-4a18-ace3-57dfd7d818a7)

**Comment by sreedharm07** [comment, marked as answer]
Hi @palsachennaiah 
There is no html in /usr/share/nginx
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/40761caf-8c79-437e-a394-4c43192a2edb)
